lms/djangoapps/django_comment_client/utils.py

```diff
@@ -805,9 +805,6 @@ def is_discussion_enabled(course_id):
     """
     Return True if Discussion is enabled for a course; else False
```

Comment:
@mtyaka If `ENABLE_DISCUSSION_SERVICE` is a global setting (not a course-specific one), it might make sense to update this docstring to something like "Return True if discussions are enabled; else False".